sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/interpreter/operator/date/BeamSqlTimestampMinusIntervalExpressionTest.java

```diff
@@ -67,7 +66,7 @@ public void testBasicProperties() {
     BeamSqlTimestampMinusIntervalExpression minusExpression =
         minusExpression(SqlTypeName.INTERVAL_DAY_MINUTE, TIMESTAMP, INTERVAL_3_MONTHS);
```

Comment:
This test case is wrong. The resulting `minusExpression` will fail on `accept()`. You should replace `INTERVAL_DAY_MINUTE` with `TIMESTAMP` for it to be correct.